ubuntu/scripts/update.sh

```diff
@@ -23,6 +24,4 @@ APT::Periodic::Enable "0";
 EOF
 
 # Upgrade all installed packages incl. kernel and kernel headers
-apt-get -y dist-upgrade;
-reboot;
-sleep 60;
+apt-get -y dist-upgrade -o Dpkg::Options::="--force-confnew";
```

Comment:
Do we need to do this? there shouldn't be any conflicts this early on since nothing else should be changed.

Reply:
Totally agree it's safe, just wanted to make sure you hit something that required it (which makes sense)